common/src/app/common/types/component.cljc

```diff
@@ -83,18 +83,19 @@
    :layout-grid-dir         :layout-container
    :layout-grid-rows        :layout-container
    :layout-grid-columns     :layout-container
-   :layout-grid-cells       :layout-container
-
-   :layout-item-margin      :layout-item
```

Comment:
I'll name this `layout-item-attrs`. It's likely that the list will be used at some moment to more things than keeping in the swap operation.